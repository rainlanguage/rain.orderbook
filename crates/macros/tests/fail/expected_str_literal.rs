#[macro_use]
extern crate rain_orderbook_macros;

struct TestStruct;

#[impl_wasm_exports]
impl TestStruct {
    #[wasm_export(unchecked_return_type = string)]
    pub async fn some_static_method(arg: String) -> Result<String, Error> {
        Ok(String::new())
    }
}

fn main() {}
