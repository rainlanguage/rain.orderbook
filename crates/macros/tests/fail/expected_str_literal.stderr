error: expected string literal
 --> tests/fail/expected_str_literal.rs:8:19
  |
8 |     #[wasm_export(unchecked_return_type = string)]
  |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
